### PR TITLE
Include git commit hash in documentation header

### PR DESCRIPTION
**Pull request by dalg24**
Tentative fix for https://github.com/ORNL/TASMANIAN/issues/417#issuecomment-541971636

### CODE REVIEW

File: Doxygen/CMakeLists.txt [hunk 1]
```diff
@@ -22,7 +22,7 @@ set(DOXYGEN_HTML_COLORSTYLE_GAMMA  "90")
 set(DOXYGEN_FORMULA_FONTSIZE "14")
 
 set(DOXYGEN_PROJECT_NAME   "Toolkit for Adaptive Stochastic Modeling and Non-Intrusive ApproximatioN: ${PROJECT_NAME} v${Tasmanian_VERSION_MAJOR}.${Tasmanian_VERSION_MINOR}${Tasmanian_version_comment}")
-set(DOXYGEN_PROJECT_NUMBER "") # if project number is given, the rendered font is too small
+set(DOXYGEN_PROJECT_NUMBER "${Tasmanian_git_hash}") # if project number is given, the rendered font is too small
```
Comment (mkstoyanov) [inline Doxygen/CMakeLists.txt]: FYI: I don't like this, doesn't look good at all. If you want the hash on-line, I would put it somewhere else, e.g., after the page link:
```
Documentation: development (rolling, currently at hash 98ae1315be4d64925284e5b28075e769f5db46fb)
```

Comment (dalg24) [reply Doxygen/CMakeLists.txt]: I am fine with that.  Also I think `Tasmanian_git_hash` was not defined because I can't see it on the doc.

Comment (mkstoyanov) [reply Doxygen/CMakeLists.txt]: See #438

**Comment by mkstoyanov** [comment]
Addressed by #438

**Pull request closed without merging**